src/lib/menu/menu-directive.ts

```diff
@@ -46,14 +46,18 @@ export class MdMenu implements AfterContentInit, MdMenuPanel, OnDestroy {
 
   positionX: MenuPositionX = 'after';
   positionY: MenuPositionY = 'below';
+  overlayTrigger = true;
 
   @ViewChild(TemplateRef) templateRef: TemplateRef<any>;
   @ContentChildren(MdMenuItem) items: QueryList<MdMenuItem>;
 
   constructor(@Attribute('x-position') posX: MenuPositionX,
-              @Attribute('y-position') posY: MenuPositionY) {
+              @Attribute('y-position') posY: MenuPositionY,
+              @Attribute('overlay-trigger') trigger = true) {
```

Comment:
Have you considered using the `public` keyword instead?  That way you don't have to explicitly set it again on line 60.

```
@Attribute('overlap-trigger') public overlapTrigger = true
```